app/angular/AngularView/src/app/booking-details/booking-details.component.html

```diff
@@ -1,33 +1,5 @@
-<style>
-    .column-left {
-        float: left;
-        width: 40%;
-    }
-
-    .column-right {
-        float: right;
-        width: 55%;
-    }
-
-    .row:after {
-        content: "";
-        display: table;
-        clear: both;
-    }
-
-    mat-list-item {
-        margin-top: 1%;
-        height: auto !important;
-        white-space: pre-wrap;
-    }
-
-    mat-checkbox.mat-checkbox-layout {
-        white-space: normal !important;
-    }
-</style>
-
-<mat-horizontal-stepper #stepper [linear]=true style="background-color: hsla(0, 0%, 97%, 0); width:80%; margin: auto;"
-    (selectionChange)="selectionChange($event)">
+<mat-horizontal-stepper class= "container" #stepper [linear]=true (selectionChange)="selectionChange($event)">
```

Comment:
preferably, have some padding between the edge of the screen and the form: the form looks too wide (that's why i initially put it as 80% of screen width, though i believe u are capable enuf to find alternatives to this 😉)